source/extensions/filters/network/dubbo_proxy/message_impl.h

```diff
@@ -0,0 +1,55 @@
+#pragma once
+
+#include "extensions/filters/network/dubbo_proxy/message.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace NetworkFilters {
+namespace DubboProxy {
+
+struct ContextBase : public Context {
```

Comment:
Why not use class? this class is not a simple POD type, also contains some virtual method

Reply:
Because it's simple, you don't need to actively add public declarations.

Reply:
I think here should use the class, this is not a simple POD type. Can't go to easy access to the data members of the break of object-oriented rules

Reply:
done